### ISSUE TITLE
Installing two groups on single host

### ISSUE THREAD

**Issue by mattkasun** [gravitl/netmaker#34]
I think this is a valid use case, i.e. one network for main job and a second network for a side gig or personal use.

- Group1 - wirecat 10.100.100.0/24 endpoint 5555 interface wc-wirecat
- Group2 - skylink 192.168.1.0/24 endpoint 5000 interface skylink

installing the second group goes smoothly and does not cause any errors, however the installation is messed up.
The interface for the second group is not picked up correctly.
The interface in /etc/netclient/.netconfig remains as wc-wirecat.  
 ```
root@aws:/etc/netclient# cat .netconfig 
server:
    address: 10.0.0.2:50051
    accesskey: 31bg2X7gVaE0oEKa
node:
    name: node-rp0lm
    interface: wc-wirecat
    group: skylink
    password: $2a$05$nBaRG3y4rcM4/yWtBCYRm.UWAjxCj8e9S6sMT80ysJAMO3CMM8jhe
    macaddress: 0a:fb:bb:72:12:92
    localaddress: 172.31.45.113
    wgaddress: 192.168.1.2
    roamingoff: false
    postup: ""
    preup: ""
    port: 5555
    keepalive: 20
    publickey: aeWeT9TRmXmuRS/916+nC4yyUTkCWwBzs9VtDE8rbgM=
    privatekey: ""
    endpoint: 18.224.213.149
    postchanges: "false"
```

As a result a second ip address is added to the wirecat tunnel.

```
19: wc-wirecat: <POINTOPOINT,NOARP,UP,LOWER_UP> mtu 1420 qdisc noqueue state UNKNOWN group default qlen 1000
    link/none 
    inet 10.100.100.3/24 scope global wc-wirecat
       valid_lft forever preferred_lft forever
    inet 192.168.1.2/24 scope global wc-wirecat
       valid_lft forever preferred_lft forever

```
connection on the second network is possible 
```
root@aws:/etc/netclient# ping 192.168.1.1
PING 192.168.1.1 (192.168.1.1) 56(84) bytes of data.
64 bytes from 192.168.1.1: icmp_seq=6 ttl=64 time=19.5 ms
64 bytes from 192.168.1.1: icmp_seq=7 ttl=64 time=19.3 ms
64 bytes from 192.168.1.1: icmp_seq=8 ttl=64 time=19.1 ms
^C
--- 192.168.1.1 ping statistics ---
9 packets transmitted, 3 received, 66.6667% packet loss, time 8123ms
rtt min/avg/max/mdev = 19.133/19.331/19.519/0.157 ms

```

but not on the first, as the wg keys are incorrect(ie they are for the second network)

```
root@aws:/etc/netclient# ping 10.100.100.1
PING 10.100.100.1 (10.100.100.1) 56(84) bytes of data.
From 10.100.100.3 icmp_seq=1 Destination Host Unreachable
ping: sendmsg: Required key not available
From 10.100.100.3 icmp_seq=2 Destination Host Unreachable
ping: sendmsg: Required key not available
^C
--- 10.100.100.1 ping statistics ---
2 packets transmitted, 0 received, +2 errors, 100% packet loss, time 1025ms
```

**Comment by mattkasun** [comment]
A "known issues" page is a great idea.

**Comment by afeiszli** [comment]
Closed issue with merge #63. See issue #56 for more details.
